unit-tests/src/test/java/org/eclipse/collections/impl/string/immutable/CodePointAdapterTest.java

```diff
@@ -401,6 +401,12 @@ public void toReversed()
         super.toReversed();
         Assert.assertEquals("cba", CodePointAdapter.adapt("abc").toReversed().toString());
     }
+/*
```

Comment:
Remove the comment?

Reply:
test is inherited from `AbstractIntIterableTestCase`